xapian-applications/omega/omegatest

```diff
@@ -432,6 +432,15 @@ printf '$last$if{$cgi{ERR},$seterror{$cgi{ERR}}}$msize$if{$error,!$error!}' > "$
 testcase '11' P=text
 testcase '11!boo!' P=text ERR=boo
 
+# Feature tests for $hash{}
+printf '$hash{$cgi{text},md5}' > "$TEST_TEMPLATE"
+testcase '098f6bcd4621d373cade4e832627b4f6' text="test"
+testcase 'b4c216e4da73d1d01277ef46d0514821' text="simple query"
+testcase 'd41d8cd98f00b204e9800998ecf8427e' text=""
```

Comment:
I think it'd be useful to keep the testcase for `" test "` or whatever it was that was here before, to confirm that whitespace normalisation is not happening (i.e. that it gives a different output to `"test"`).